readme.md

```diff
@@ -180,10 +214,14 @@ Once installation is working, try creating and requesting resources. Note that t
 
 * **Your Answer:**
 
+It created an _id field for the actor.
+
 * **Question:** With the current routes that we have, how would you upate the name of a character in a series?
 
 * **Your Answer:**
 
+use the PUT route to update a character.
```

Comment:
You could either modify the PUT route or create a new one. Either way, you'll need to go _through_ the Series model to update the Character.